extra/markdown-extra/ErusevMarkdownExtra.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Extra\Markdown;
+
+class ErusevMarkdownExtra implements MarkdownInterface
```

Comment:
Why creating a new ErusevMarkdownExtra ? The existing ErusevMarkdown class supports ParsedownExtra already.